framework/src/task_manager/README.md

```diff
@@ -0,0 +1,71 @@
+# Task Manager
+
+Task manager manages the registered applications.  
+If management between applications is needed, by using the task manager, an application can easily manage other applications.  
+- Example cases  
+ : Bixby can start the Alarm application through the task manager.  
+ : When Wi-Fi disconnection message is broadcasted, applications can receive the broadcast message.
+
+## Features
+### API List
+| Functionality | APIs                               |
+|---------------|------------------------------------|
+| Registration  | task_manager_register()            |
+|               | task_manager_unregister()          |
+| Control       | task_manager_start()               |
+|               | task_manager_restart()             |
+|               | task_manager_terminate()           |
+|               | task_manager_termination_cb()      |
+|               | task_manager_pause()               |
+|               | task_manager_resume()              |
+| Message       | task_manager_unicast()             |
+|               | task_manager_broadcast()           |
+|               | task_manager_set_unicast_cb()      |
+|               | task_manager_set_broadcast_cb()    |
+| Information   | task_manager_getinfo_with_name()   |
+|               | task_manager_getinfo_with_handle() |
+|               | task_manager_getinfo_with_group()  |
+|               | task_manager_clean_info()          |
+|               | task_manager_clean_infolist()      |
+
+Please find details in [Header file](#header-file).
+
+### Permission
+All registered applications have their own permission, and the task manager checks the permission for every management request.  
+If controller application does not have proper permission for controllee applications, it can't manage the controllee applications.  
+- Permission types
+	- TM_TASK_PERMISSION_ALL : Any Task can request the control to Task Manager  
+	- TM_TASK_PERMISSION_GROUP : Only same group task can request the control to Task Manager  
+		(*group : Tasks set which registered from same parent task)  
+	- TM_TASK_PERMISSION_DEDICATE : Only a task which requested to register that task can request the control to Task Manager
+
+### State
+Each Application can have four states.  
```

Comment:
Task Manager manages each application with four states, unregistered, stop, running and paused.
Please find below how to move each state by APIs.